### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by chrsc59** [MyElectricalData/myelectricaldata_import#256] [Q&A]
Bonjour, 
Je cherchais à faire fonctionner l'intégration HomeAssistant myEnedis/apiEnedis qui ne remonte plus rien depuis longtemps, et je suis donc tombé sur myelectricaldata, après avoir vu l'option dans la configuration, en remplacement de enedisgateway.tech, mais qui ne marche pas mieux.

Mais rien ne veut marcher, mais bien que je comprenne bien que tout çà est encore en développement,  je pose quand même ici la question: 

Pouvez-vous clarifier dans la doc en introduction

- Ce que contient ce dépot : une intégration? un addon ?
- Compatible avec quoi ? HASSIO? HAOS? 
- Installable comment ? HACS? Manuel ? Docker only ?

Ca serait sympa pour les gens comme moi qui ne suivent pas l'affaire depuis longtemps, mais qui veulent bien s'y mettre
Merci

**Comment by m4dm4rtig4n** [comment]
Hello,

myEnedis / apiEnedis utilise ttes les 2 enedisgateway.tech (ma passerelle qui fait passe plat entre les particuliers & Enedis).
MyElectricalData est une nouvelle version qui vient remplacer d'EnedisGateway qui elle sera couper courant Mai 2023.

Ici c'est le dépot du client MyElectricalData qui permet de récupérer les information d'Enedis via la passerelle dédié https//www.myelectricaldata.fr

Je suis en train de tt metre dans le wiki qui est en cours de redaction :)
https://github.com/m4dm4rtig4n/myelectricaldata/wiki/02.-Installation

Sinon tu as 2 options d'installation : 
- Tu peut installer en standalone via un simple conteneur Docker.
- Directement sur HASSIO en addon via le dépot d'Alexbelgium => https://github.com/alexbelgium/hassio-addons

**Comment by chrsc59** [comment]
Merci, J'ai finalement pu rajouter le dépot d'Alexbelgium  et installer l'addon

Je pense tout de même qu'il serait important d'adapter les documentations aux noobs, et surtout aux versions actuelles de HomeAssistant.

Je suis rentré il y peu de temps dans le monde HA, et il m'a semblé en particulier que les types d'installations (HAOS, Core, Supervised, VM, Docker) étaient difficiles à choisir, que le terme "HASSIO" n'existe plus depuis quelques temps, que le chemin vers l'ajout de dépots d'intégrations HACS et l'ajout d'addons n'est pas si évidente dans le GUI, etc...

Merci pour l'aide, bravo pour le travail déjà fait et bon courage pour le dev

**Reply by m4dm4rtig4n** [reply]
Je viens d'ajouter un peu de doc à ce sujet :
https://github.com/m4dm4rtig4n/myelectricaldata/wiki/02.-Installation

Après il néxiste pas vraiment de mode "Noob" malheuresement (à part l'addon sur HA).
